python_modules/dagster/dagster/_daemon/backfill.py

```diff
@@ -10,6 +15,26 @@
 from dagster._utils.error import SerializableErrorInfo
 
 
+@contextmanager
+def _get_instigation_logger_if_env_var_set(
+    instance, backfill_id: str, default_logger: logging.Logger
+):
+    if os.getenv("STORE_BACKFILL_LOGS", None):
+        evaluation_time = pendulum.now("UTC")
+        log_key = ["backfill", backfill_id, evaluation_time.strftime("%Y%m%d_%H%M%S")]
+        with InstigationLogger(
```

Comment:
the `InstigationLogger` always uses the `dagster` logger, whereas the backfill daemon uses the backfill-daemon-logger https://github.com/dagster-io/dagster/blob/97c80ef86a5c077eafe007d0af87d7751e08a604/python_modules/dagster/dagster/_core/definitions/instigation_logger.py#L92


likely need to make the name of the logger a parameter to instigation logger or something